backend/api/pipeline.proto

```diff
@@ -320,6 +320,15 @@ message Pipeline {
   repeated ResourceReference resource_references = 9;
 }
 
+enum PipelineSpecFormat {
+  PIPELINE_SPEC_UNSPECIFIED = 0;
```

Comment:
Should we assume that PIPELINE_SPEC_UNSPECIFIED is also V1 pipeline? (argo workflow)

Reply:
I've been debating with myself for this too. I think the answer is yes, probably we don't even need the `PIPELINE_SPEC_UNSPECIFIED` enum then.

Reply:
actually I made a typo, the value should be `PIPELINE_SPEC_FORMAT_UNSPECIFIED`

Reply:
Got it, thank you for confirming! Then I will assume it is V1 pipeline if this enum doesn't exist in response.